### ISSUE TITLE
Update playbook to use v1.5.x of Asciidoctor repository

### ISSUE THREAD

**Issue by graphitefriction** [asciidoctor/docs.asciidoctor.org#15]
Once the v1.5.7-docs branch is merged into the v1.5.x and master branches, the production playbook needs to be updated to aggregate the Asciidoctor and AsciiDoc components from the correct branch.

Also, do we want to run a separate development playbook for the future release documentation or use a different strategy?

**Comment by graphitefriction** [comment]
@mojavelinux Since the docs.asciidoctor.org production infrastructure and Asciidoctor 2.0.0 beta release will roughly coincide, do you just want to promote the docs only to dev (and therefore 2.0) and just skip 1.5.x?

**Comment by mojavelinux** [comment]
To manage the docs correctly, we really need to be on semantic versioning. So that's the case right there for just aligning the new docs site with the first 2.0.x release.

Here's the policy that I think will work:

* There's no separate docs branch; the docs are co-hosted with the code
* The repository has branches that each correspond to a minor release (v2.0.x, v2.1.x, v2.2.x, v3.0.x, etc.).
* These branches host the docs for that minor version.
* These branches are also the origin of patch releases, which become tags (v2.0.0, v2.0.1, etc.)
* The docs version consists of the first two positions (2.0, 2.1, 2.2, 3.0, etc.)
* The master branch holds the next minor release (until it's determined that it needs to be the next major release)
* The docs for the next minor release doesn't show up in the production site; it may be in a preview site (so master is not in the production playbook)
* A software release should always be done from a branch. Before the first release of a minor version, that branch should be created. That allows a release to be spawned, but also makes the docs available.
* The version number in the antora.yml file of the component should be updated when the software is released (just like updating the VERSION constant)

**Comment by mojavelinux** [comment]
cc: @Mogztter

**Comment by Mogztter** [comment]
> There's no separate docs branch; the docs are co-hosted with the code

:100: 
And I agree with the other items :+1:

**Comment by graphitefriction** [comment]
We've moved beyond 1.5.x so this issue isn't relevant any more
